Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/RectangularBeamProfile.h

```diff
@@ -30,6 +30,15 @@ class MANTID_ALGORITHMS_DLL RectangularBeamProfile final : public IBeamProfile {
   IBeamProfile::Ray generatePoint(Kernel::PseudoRandomNumberGenerator &rng,
                                   const Geometry::BoundingBox &bounds) const override;
   Geometry::BoundingBox defineActiveRegion(const Geometry::BoundingBox &) const override;
+  /// Returns the min point of the profile
+  inline Kernel::V3D minPoint() const { return Kernel::V3D{m_min[0], m_min[1], m_min[2]}; }
```

Comment:
I noticed there is a `m_upIdx`, `m_beamIdx` and `m_horIdx`. Just thought I'd ask whether they can be used here instead of 0, 1 and 2?

Reply:
I don't think we can do this although it is something I also wondered about.

I think the m_upIdx and m_horIdx are used to indicate which of the x, y, z axes are up and horizontal. It's normally y up and x horizontal but some instruments use a different convention. The benefit of these fields (I think) is that a user of the RectangularBeamProfile class can call the constructor with a width and a height without needing to know which of the axes corresponds to the width\height.

I think regardless of this choice, the 0th field in m_min always contains a value along the x axis. And the developer calling minPoint() would expect the first (x) element in the returned V3D object to contain an x axis coordinate. So I don't think we can use m_upIdx\m_horIdx here